Ecosystem Partners/Marketplace Guides/getting-started-with-microsoft-tfs.md

```diff
@@ -0,0 +1,178 @@
+
+After reading this article, the user should be able to deploy a TFS 2013 blueprint to create a TFS 2013 server.
+
+### Prerequisite
+
+Access to the CenturyLink Cloud platform as an authorized user.
+
+#### Steps to Deploy Blueprint
+
+
+1. **Locate the "Team Foundation Server" Blueprint**
+
+  1. Starting from the CenturyLink Control Panel, navigate to the Blueprints Library.
+  2. Search for “TFS” in the keyword search on the right side of the page.
+  3. Locate the specific "Team Foundation Server" Blueprint that you require (see below)
+
+![screenshot 1](../../images/MSFT_TFS/TFSBP.png)
+
+2. **Choose and Deploy the Blueprint.**
+
+  1. Click the "Team Foundation Server" Blueprint that you require.
+
+3. **Customize the Blueprint**
+
+  1. **Build Server**
+
+        1. Set server options including password.
+
+		![screenshot 2](../../images/MSFT_TFS/TFS1B.png)
+
+  2. **Set Server Name**
+  
+        1. Set the required server name.
+		
+![screenshot 3](../../images/MSFT_TFS/TFS2B.png)
+
+  3. **Specify Credentials**
+  
+        1. If required, set the server credentials.
+
+![screenshot 4](../../images/MSFT_TFS/TFS3B.png)
+
+
+  4. **Select SQL Version ("TFS with SQL Server" only)**
+  
+        1. Select either 2014 Standard or 2014 Enterprise.
+
+![screenshot 5](../../images/MSFT_TFS/TFS4B.png)
+
+  5. **Select SQL Features ("TFS with SQL Server" only)**
+  
+        1. Select the options as shown.
+
+![Microsoft logo](../../images/MSFT_TFS/TFS5B.png)
+
+4. **Review and Confirm the Blueprint**
+  1. Click “next: step 2”
+  2. Verify your configuration details.
+
+5. **Deploy the Blueprint**
+  1. Once verified, click on the ‘deploy blueprint’ button. You will see the deployment details along with an email stating the Blueprint is queued for execution.
+  2. This will kick off the blueprint deploy process and load a page to allow you to track the progress of the deployment.
+
+6. **Monitor the Activity Queue**
+  1. Monitor the Deployment Queue to view the progress of the blueprint.
+  2. You can access the queue at any time by clicking the Queue link under the Blueprints menu on the main navigation drop-down.
+  3. Once the blueprint completes successfully, you will receive an email stating that the blueprint build is complete. Please do not use the application until you have received this email notification
+
+
+### Access your TFS 2013 server
+
+To access your TFS 2013 server from a computer outside the CenturyLink Cloud network use the OpenVPN client as specified here: [How To Configure Client VPN](https://www.ctl.io/knowledge-base/network/how-to-configure-client-vpn/)
+
+### Pricing
+
+The costs associated with this blueprint deployment include CenturyLink Cloud infrastructure usage.  
+
+Additional licensing costs will apply for Microsoft software, including TFS 2013 and SQL Server. If you deploy with SQL Server, charges will apply for SQL Server.  These will be shown in the indicated cost for the blueprint when you deploy it.  
```

Comment:
this is not factual.  the blueprint engine does not show software costs such as SQL server in the cost estimates.